pkg/storage/gcp/getter.go

```diff
@@ -0,0 +1,48 @@
+package gcp
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/gomods/athens/pkg/config"
+	"github.com/gomods/athens/pkg/storage"
+)
+
+// Get retrieves a module at a specific version from storage as a (./pkg/storage).Version
+//
+// The caller is responsible for calling close on the Zip ReadCloser
+func (s *Storage) Get(module, version string) (*storage.Version, error) {
```

Comment:
If we access the files from CDN do we even need that?

Reply:
As above, not sure. Just implementing the interfaces provided by the other storage packages. Will we be removing this?

Reply:
For CDN is not needed, if someone wants a proxy with this in place, it might be helpful